### DISCUSSION TITLE
Update 1.15.5 to 1.17.1

### DISCUSSION THREAD

**Discussion by forenade** [kimai/kimai#3162] [On-Premise: Installation & Updates]
Iam running a kimai 1.15.5 server now for a few months now and it have worked great. Big thanks for your work. 

Now to my problem ..
With the new 1.17 i have seen some great plugins that i wont to buy and add to my kimai server så started to try updating it. 

First i made a backup then i started to try update it like the guid show but that gave my tons of errors on update. 
So i tryed installing a new 1.17.1 server on ubuntu server and that worked great but when i then tryed to restore the old server data with easy backup plugin kimai gave me database error. 

I have used the same name for database and same user and password as the old server but still the database say that i dont have acess. 

Do someone have a cloue what iam doing wrong ?

**Comment by kevinpapst** [comment]
This might help: https://www.kimai.org/documentation/backups.html

"Tons of error" is a highly unspecific description. 

Updating Kimai shouldn't be a problem, but it might feel complicated if you have no experience with the tools used.
I can help, but next time please provide more details while being in the process - not afterwards.
